easy-random-randomizers/src/main/java/org/jeasy/random/randomizers/PasswordRandomizer.java

```diff
@@ -0,0 +1,185 @@
+ *   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ */
+package org.jeasy.random.randomizers;
+
+import org.jeasy.random.api.Randomizer;
+
+import java.util.Locale;
+
+/**
+ * A {@link Randomizer} that generates random password.
+ *
+ * 
+ */
+public class PasswordRandomizer extends FakerBasedRandomizer<String> {
+	private boolean includeUppercase,includeSpecial;
+	private int min,max;
+	
+
+
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     */
+    public PasswordRandomizer() {
+    }
+
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @param seed the initial seed
+     */
+    public PasswordRandomizer(long seed) {
+        super(seed);
+    }
+
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @param seed   the initial seed
+     * @param locale the locale to use
+     */
+    public PasswordRandomizer(final long seed, final Locale locale) {
+        super(seed, locale);
+    }
+
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @param seed   the initial seed
+     * @param min	 the minimum character of password
+     * @param max	 the maximum character of password
+     */
+    public PasswordRandomizer(final long seed,  final int min,final int max) {
+        super(seed );
+        this.min = min;
+        this.max=max;
+    }
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @param seed   the initial seed
+     * @param min	 the minimum character of password
+     * @param max	 the maximum character of password
+     * @param includeUppercase   true to generate password contains Uppercase Character , false otherwise
+     */
+    public PasswordRandomizer(final long seed,  final int min,final int max ,final boolean includeUppercase) {
+        super(seed );
+        this.min = min;
+        this.max=max;
+        this.includeUppercase=includeUppercase;
+    }
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @param seed   the initial seed
+     * @param min	 the minimum character of password
+     * @param max	 the maximum character of password
+     * @param includeUppercase   true to generate password contains Uppercase Character , false otherwise
+     * @param includeSpecial   true to generate password contains Special Character{'!', '@', '#', '$', '%', '^', '&', '*'} , false otherwise
+     */
+    public PasswordRandomizer(final long seed,  final int min,final int max ,final boolean includeUppercase,final boolean includeSpecial) {
+        super(seed );
+        this.min = min;
+        this.max=max;
+        this.includeUppercase=includeUppercase;
+        this.includeSpecial=includeSpecial;
+    }
+
+    /**
+     * Create a new {@link PasswordRandomizer}.
+     *
+     * @return a new {@link PasswordRandomizer}
+     */
+    public static PasswordRandomizer aNewPasswordRandomizer() {
```

Comment:
Could you please remove these static factory methods? They were deprecated in other places (see 735b2450d4bb775f144a4dcb8349824982ad50d5)